ext/ldap/ldap.c

```diff
@@ -75,18 +76,26 @@ typedef struct {
 #if defined(LDAP_API_FEATURE_X_OPENLDAP) && defined(HAVE_3ARG_SETREBINDPROC)
 	zval rebindproc;
 #endif
+	zend_object std;
 } ldap_linkdata;
 
+typedef struct {
+	LDAPMessage *result;
+	zend_object std;
+} ldap_resultdata;
+
 typedef struct {
 	LDAPMessage *data;
 	BerElement  *ber;
 	zval         res;
```

Comment:
I think the `res` field is not used anymore, but at least I can't find it any read operations. Can I then remove it?

Reply:
Sure :)

Reply:
I got an assertion error as soon as i removed it, so i must have missed something.

Reply:
Ah, it looks like this is used to make sure the result is not dropped before the resultentry. So yeah, it's necessary.